lib/routes/user/home/account_page.dart

```diff
@@ -164,6 +165,13 @@ class AccountPageState extends State<AccountPage> with SingleTickerProviderState
               return account != null && !account.initial ? FloatingActionsBar(account, height, heightFactor) : Positioned(top: 0.0, child: SizedBox());
             },
           ),
+          ScrollWatcher(
+            controller: widget.scrollController,
+            builder: (context, offset) {
+              double height = (DASHBOARD_MAX_HEIGHT - offset).clamp(DASHBOARD_MIN_HEIGHT, DASHBOARD_MAX_HEIGHT);
+              return account != null && !account.initial ? InvoiceBottomSheet(_invoiceBloc, height < 160.0) : Positioned(top: 0.0, child: SizedBox());
```

Comment:
We need to show the invoice sheet only if there is an opened channel, the same as we do with the corresponding menu items. (` if account.active` )